packages/spruce-skill/config/default.js

```diff
@@ -29,7 +30,10 @@ module.exports = {
 	METRICS_SEQUELIZE_DISABLED: process.env.METRICS_SEQUELIZE_DISABLED === 'true',
 	API_HOST: process.env.API_HOST,
 	API_KEY: process.env.API_KEY,
-	SKILL_STYLESHEET: process.env.SKILL_STYLESHEET,
+	SKILL_STYLESHEET:
+		process.env.SKILL_STYLESHEET ||
+		`https://CHANGE_ME_CDN_URL/stylesheets/${HEARTWOOD_VERSION ||
+			'latest'}/heartwood-components.min.css`,
```

Comment:
I'm assuming we'll put a cache on the CDN  so `latest` will run into issues there; I wonder if we should also hash the project for something like:

```
`https://${CDN_HOST}/stylesheets/heartwood-components-${HEARTWOOD_VERSION}-${HEARTWOOD_HASH}.min.css`
```